x-pack/plugins/fleet/common/openapi/paths/service_tokens.yaml

```diff
@@ -0,0 +1,24 @@
+post:
+  summary: Generate service tokens
+  tags: []
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+  operationId: generate-service-token
+  parameters:
+    - $ref: ../components/headers/kbn_xsrf.yaml
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
```

Comment:
`{name, value}` is actually the response body, not the request body. Same for the deprecated one.